lib/tasks/db.rake

```diff
@@ -89,6 +92,12 @@ end
   end
 
   def config
+    require 'db/connection'
```

Comment:
It's safe, but I do find that loading unnecessary code slows things down, and I think that on principle we should avoid it.